### PR TITLE
JSONP client parity

### PR DESCRIPTION
**Pull request by davidfischer**
- Support keyword targeting
- Support campaign-type restriction

### CODE REVIEW

File: adserver/api/views.py [hunk 1]
```diff
@@ -74,6 +74,8 @@ class AdDecisionView(GeoIpMixin, APIView):
             The number and order must correspond to the ``div_ids``.
         :<json string priorities: An optional ``|`` delimited string of priorities for different ad types.
             The number and order matter, applying to ``div_ids`` and ``ad_types``.
+        :<json array keywords: An optional ``|`` delimited string of keywords.
+        :<json array campaign_types: An optional ``|`` delimited string of campaign types.
```
Comment (ericholscher) [inline adserver/api/views.py:78]: We should probably define these concepts somewhere (eg. what's a campaign type? Keyword?), if we haven't. I had a user ask what they are recently when I linked them to this.

File: adserver/api/views.py [hunk 2]
```diff
@@ -74,6 +74,8 @@ class AdDecisionView(GeoIpMixin, APIView):
             The number and order must correspond to the ``div_ids``.
```
Comment (ericholscher) [inline adserver/api/views.py:74]: We should probably remove the mention of `Supports the same parameters as via POST with the following changes:` now I think, and just include the full content here?

**Comment by davidfischer** [comment]
I updated the documentation and filled out some details and the GET request section specifically.

**Comment by davidfischer** [comment]
I do think a lot of these examples should go into the https://github.com/readthedocs/ethical-ad-client though since that's where it'll be used.

**Comment by ericholscher** [comment]
Docs changes look great. I agree we want more of this content on the client as well 👍

**Pull request merged**